shelldock-emrakul.md

```diff
@@ -128,11 +129,21 @@ post-Doomsday graveyard. And that's not including the cards you will
 draw in the turn(s) that you attack with Emrakul. So this is probably 
 not feasible in practice.
 
+One nice thing is that if you get the Shelldock Isle in hand, if the opponent
+holds a Surgical Extraction or draws into it for their turn then it does nothing
+to disrupt the pile as Hideaway digs four cards deep thereby encompassing the whole 
+pile. Be warned however as if you do decide to include a secondary Doomsday
+in the pile they can extract it off of the one in your Graveyard thereby reducing 
+the number of turns you have before decking yourself by one. The presence of a
+Conjurer's Bauble can help remove this risk by replacing the targeted spell back
+into the deck.
```

Comment:
There is an issue here as when you target the DD in your graveyard with CB, your opponent may use Surgical targeting the DD in your graveyard, which will not only shuffle your library but also make the draw ability of the CB fizzle, because it's target is no longer legal.